### DISCUSSION TITLE
install imagemagick over ssh

### DISCUSSION THREAD

**Discussion by Big-Chair** [ImageMagick/ImageMagick#2690] [Help]
following this guide 
https://www.ionos.com/help/server-cloud-infrastructure/managed-dedicated-server/install-imagemagick-over-ssh/

$ make install

but then get error 
/usr/bin/install: cannot create regular file '/usr/local/lib/libMagickCore-7.Q16HDRI.so.7.0.0': Read-only file system

not sure what to do next?

  CXX      Magick++/lib/libMagick___7_Q16HDRI_la-TypeMetric.lo
  CXXLD    Magick++/lib/libMagick++-7.Q16HDRI.la
cp -f MagickCore/ImageMagick.pc MagickCore/ImageMagick-7.Q16HDRI.pc
cp -f MagickCore/MagickCore.pc MagickCore/MagickCore-7.Q16HDRI.pc
cp -f MagickWand/MagickWand.pc MagickWand/MagickWand-7.Q16HDRI.pc
cp -f Magick++/lib/Magick++.pc Magick++/lib/Magick++-7.Q16HDRI.pc
make[1]: Leaving directory '/homepages/43/d76114616/htdocs/ImageMagick-7.0.10-33'
(uiserver):u34591154:~/ImageMagick-7.0.10-33$ make install
make  install-am
make[1]: Entering directory '/homepages/43/d76114616/htdocs/ImageMagick-7.0.10-33'
make[2]: Entering directory '/homepages/43/d76114616/htdocs/ImageMagick-7.0.10-33'
 /usr/bin/mkdir -p '/usr/local/lib'
 /bin/bash ./libtool   --mode=install /usr/bin/install -c   MagickCore/libMagickCore-7.Q16                          HDRI.la MagickWand/libMagickWand-7.Q16HDRI.la Magick++/lib/libMagick++-7.Q16HDRI.la '/usr/                          local/lib'
libtool: install: /usr/bin/install -c MagickCore/.libs/libMagickCore-7.Q16HDRI.so.7.0.0 /u                          sr/local/lib/libMagickCore-7.Q16HDRI.so.7.0.0
/usr/bin/install: cannot create regular file '/usr/local/lib/libMagickCore-7.Q16HDRI.so.7.                          0.0': Read-only file system
make[2]: *** [Makefile:6307: install-libLTLIBRARIES] Error 1
make[2]: Leaving directory '/homepages/43/d76114616/htdocs/ImageMagick-7.0.10-33'
make[1]: *** [Makefile:12363: install-am] Error 2
make[1]: Leaving directory '/homepages/43/d76114616/htdocs/ImageMagick-7.0.10-33'
make: *** [Makefile:12357: install] Error 2
(uiserver):u34591154:~/ImageMagick-7.0.10-33$ pwd
/kunden/homepages/43/d76114616/htdocs/ImageMagick-7.0.10-33
(uiserver):u34591154:~/ImageMagick-7.0.10-33$ ldconfig /usr/local/lib
-bash: ldconfig: command not found
(uiserver):u34591154:~/ImageMagick-7.0.10-33$ ldconfig /usr/local/lib
-bash: ldconfig: command not found
(uiserver):u34591154:~/ImageMagick-7.0.10-33$ ldconfig /usr/local/lib
-bash: ldconfig: command not found
(uiserver):u34591154:~/ImageMagick-7.0.10-33$ /usr/local/bin/convert logo: logo.gif
-bash: /usr/local/bin/convert: No such file or directory
(uiserver):u34591154:~/ImageMagick-7.0.10-33$ ^C
(uiserver):u34591154:~/ImageMagick-7.0.10-33$ pwd
/kunden/homepages/43/d76114616/htdocs/ImageMagick-7.0.10-33
(uiserver):u34591154:~/ImageMagick-7.0.10-33$ cd ImageMagick-7.0.10-33/
-bash: cd: ImageMagick-7.0.10-33/: No such file or directory
(uiserver):u34591154:~/ImageMagick-7.0.10-33$ make install
make  install-am
make[1]: Entering directory '/homepages/43/d76114616/htdocs/ImageMagick-7.0.10-33'
make[2]: Entering directory '/homepages/43/d76114616/htdocs/ImageMagick-7.0.10-33'
 /usr/bin/mkdir -p '/usr/local/lib'
 /bin/bash ./libtool   --mode=install /usr/bin/install -c   MagickCore/libMagickCore-7.Q16HDRI.la MagickWand/libMagickWand-7.Q16HDRI.la Magick++/lib/libMagick++-7.Q16HDRI.la '/usr/local/lib'
libtool: install: /usr/bin/install -c MagickCore/.libs/libMagickCore-7.Q16HDRI.so.7.0.0 /usr/local/lib/libMagickCore-7.Q16HDRI.so.7.0.0
/usr/bin/install: cannot create regular file '/usr/local/lib/libMagickCore-7.Q16HDRI.so.7.0.0': Read-only file system
make[2]: *** [Makefile:6307: install-libLTLIBRARIES] Error 1
make[2]: Leaving directory '/homepages/43/d76114616/htdocs/ImageMagick-7.0.10-33'
make[1]: *** [Makefile:12363: install-am] Error 2
make[1]: Leaving directory '/homepages/43/d76114616/htdocs/ImageMagick-7.0.10-33'
make: *** [Makefile:12357: install] Error 2
(uiserver):u34591154:~/ImageMagick-7.0.10-33$

**Comment by Big-Chair** [comment]
Trying to install imagick on php7.4 on a Linux server

**Comment by t246246** [comment]
I think you have no write permission to /usr/local/lib.  I don't know your server but if it is impossible to connect server with another user who have more access rights, you have to install ImageMagick to another directory, for example to your home directory.
